### PR TITLE
Feature/puik : POC integrate a new design system in CORE

### PR DESCRIPTION
**Pull request by cnavarro-prestashop**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop / 8.1.x / 8.0.x / 1.7.8.x
| Description?      | Please be specific when describing the PR. <br> Every detail helps: versions, browser/server configuration, specific module/theme, etc. Feel free to add more information below this table.
| Type?             | bug fix / improvement / new feature / refacto
| Category?         | FO / BO / CO / IN / WS / TE / LO / ME / PM
| BC breaks?        | yes / no
| Deprecations?     | yes / no
| How to test?      | Indicate how to verify that this change works as expected.
| Fixed ticket?     | Fixes #{issue number here}, Fixes #{another issue number here}
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

File: admin-dev/themes/puik-theme/public/theme.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import '../node_modules/@prestashopcorp/puik/dist/index.css';
```
Comment (Progi1984) [inline admin-dev/themes/puik-theme/public/theme.css:1]: :warning: It seems you use a proprietary package.

File: classes/controller/AdminController.php [hunk 2]
```diff
@@ -2640,8 +2640,9 @@ public function setMedia($isNewTheme = false)
             if ($this->isMultistoreEnabled()) {
                 $this->addJs(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/multistore_dropdown.bundle.js?v=' . _PS_VERSION_);
             }
+
             $this->addJqueryPlugin(['chosen', 'fancybox']);
-        } else {
+        }  else {
```
Comment (ghost) [inline classes/controller/AdminController.php:2645]:
```suggestion
        } else {
```

**Review by ghost — COMMENTED** [review]
Would it be possible[ to obtain information](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/) on this change?

File: classes/controller/AdminController.php (hunk 2 above)
Comment (ghost) [reply classes/controller/AdminController.php:2645]: Small feeback

File: admin-dev/themes/puik-theme/public/theme.css (hunk 1 above)
Comment (cnavarro-prestashop) [reply admin-dev/themes/puik-theme/public/theme.css:1]: Hello guys I’m just experimenting something in a POC but as soon as I’m ready I’ll tell you more about that contribution. Thanks for the review and I close the PR.

**Comment by cnavarro-prestashop** [comment]
Just a POC to test if we could use another UI KIT. Not relevant for the moment so I close the PR.

**Pull request closed without merging**